datahub-web-react/src/app/entity/EntityRegistry.tsx

```diff
@@ -128,6 +128,10 @@ export default class EntityRegistry {
                         entity: relationship.entity as EntityInterface,
                         type: (relationship.entity as EntityInterface).type,
                     })),
+                downstreamRelationships: genericEntityProperties?.downstream?.relationships
+                    ?.filter((relationship) => relationship.entity)
+                    // eslint-disable-next-line @typescript-eslint/dot-notation
+                    ?.filter((relationship) => !relationship.entity?.['status']?.removed),
```

Comment:
I'm assuming this is safe even when 'status' doesn't exist due to the ?

Reply:
indeed